lib/rules/capitalized-comments.js

```diff
@@ -0,0 +1,258 @@
+                                            }
+                                        },
+                                        additionalProperties: false
+                                    }
+                                },
+                                additionalProperties: false
+                            }
+                        },
+                        additionalProperties: false
+                    }
+                ]
+            }
+        ]
+    },
+
+    create(context) {
+
+        const normalizedOptions = getAllNormalizedOptions(context.options[0]),
+            sourceCode = context.getSourceCode();
+
+        //----------------------------------------------------------------------
+        // Helpers
+        //----------------------------------------------------------------------
+
+        /**
+         * Checks whether a comment is an inline comment.
+         *
+         * @param {ASTNode} comment The comment node to check.
+         * @returns {boolean} True if the comment is an inline comment, false
+         * otherwise.
+         */
+        function isInlineComment(comment) {
+            const firstToken = sourceCode.getFirstToken(comment),
+                previousToken = sourceCode.getTokenBefore(firstToken);
+
+            return previousToken &&
+                firstToken.loc.line === previousToken.loc.line;
+        }
+
+        /**
+         * Process a comment to determine if it needs to be reported.
+         *
+         * @param {ASTNode} comment The comment node to process.
+         * @returns {void}
+         */
+        function processComment(comment) {
+            const options = normalizedOptions[comment.type];
+
+            // 1. Check for markers.
+            if (options.markers.some(m => comment.value[0] === m)) {
+                return;
+            }
+
+            // 2. Check for default ignore pattern.
+            if (DEFAULT_IGNORE_PATTERN.test(comment.value)) {
+                return;
+            }
+
+            // 3. Check for custom ignore pattern.
+            if (options.ignorePattern && RegExp(options.ignorePattern).test(comment.value)) {
+                return;
+            }
+
+            // 4. Check for inline comments.
+            if (options.ignoreInlineComments && isInlineComment(comment)) {
+                return;
+            }
+
+            // 5. Is the initial word character a letter?
+            const commentWordCharsOnly = comment.value
+                .replace(WHITESPACE, "")
+                .replace(/\*/g, "");        // FIXME: Should this be hardcoded?
+
+            const firstWordChar = commentWordCharsOnly[0];
+
+            if (!LETTER_PATTERN.test(firstWordChar)) {
+                return;
+            }
+
+            // 6. Check the case of the initial word character.
+            const isUppercase = UPPERCASE_PATTERN.test(firstWordChar),
+                isLowercase = LOWERCASE_PATTERN.test(firstWordChar);
```

Comment:
I think we can use standard methods.

```js
const isUppercase = firstWordChar.toLocaleLowerCase() !== firstWordChar,  // if it can be converted to a lowercase character, it's an uppercase character.
    isLowercase = firstWordChar.toLocaleUpperCase() !== firstWordChar;
```